src/menu/rendersettingsmenustate.cpp

```diff
@@ -369,54 +383,31 @@ std::unique_ptr<MenuState>
       "%1%x Anisotropic Filtering", world.getEngine().getEngineConfig()->renderSettings.anisotropyLevel));
   }
 
-  {
-    int maxW = 0;
-    int totalH = 0;
-    static constexpr int Separation = 10;
-    for(const auto& listBox : m_listBoxes)
-    {
-      listBox->fitToContent();
-      maxW = std::max(maxW, listBox->getSize().x);
-      totalH += listBox->getSize().y + Separation;
-    }
-
-    int y = ui.getSize().y - totalH - 90;
-    for(const auto& listBox : m_listBoxes)
-    {
-      const auto height = listBox->getSize().y;
-      listBox->setSize({maxW, height});
-      listBox->setPosition({(ui.getSize().x - maxW) / 2, y});
-      y += height + Separation;
-    }
-  }
+  m_tabs->fitToContent();
+  m_tabs->setPosition({(ui.getSize().x) / 2 - (m_tabs->getSize().x / 2), ui.getSize().y -m_tabs->getSize().y - 90});
```

Comment:
```suggestion
  m_tabs->setPosition({(ui.getSize().x - m_tabs->getSize().x / 2), ui.getSize().y - m_tabs->getSize().y - 90});
```

Reply:
This results in an X position of 815 which is half off the screen. 
I have used 
`{ui.getSize().x - m_container->getSize().x) / 2,ui.getSize().y - m_tabs->getSize().y - 90`}
the same as in audio settings.

Reply:
yes, sorry, misplaced `)`, that should have gone after `getSize().x` in the suggestion